aspnet/mvc/mvc5.md

```diff
@@ -41,43 +41,45 @@ ASP.NET MVC now supports [attribute routing](https://blogs.msdn.com/b/webdev/arc
+Visual Studio enhanced the experience of creating new web projects, starting in Visual Studio 2013. In the **New ASP.NET Web Project** dialog you can select the project type you want, configure any combination of technologies (Web Forms, MVC, Web API), configure authentication options, add Docker support, and add a unit test project.
 
-![New ASP.NET Project](mvc5/_static/image1.png)
+![New ASP.NET Project](mvc5/_static/new-aspnet-web-app-dialog.png)
 
-The new dialog enables you to change the default authentication options for many of the templates. For example, when you create an ASP.NET Web Forms project you can select any of the following options:
+The dialog enables you to change the default authentication options for many of the templates. For example, when you create an ASP.NET Web Forms project you can select any of the following options:
 
 - No Authentication
 - Individual User Accounts (ASP.NET membership or social provider log in)
 - Organizational Accounts (Active Directory in an internet application)
 - Windows Authentication (Active Directory in an intranet application)
 
-![Authentication options](mvc5/_static/image2.png)
+![Authentication options](mvc5/_static/change-authentication-dialog.png)
 
-For more information about the new process for creating web projects, see [Creating ASP.NET Web Projects in Visual Studio 2013](../visual-studio/overview/2013/creating-web-projects-in-visual-studio.md). For more information about the new authentication options, see [ASP.NET Identity](../identity/overview/index.md).
+For more information about the process for creating web projects, see [Creating ASP.NET Web Projects in Visual Studio](../visual-studio/overview/2013/creating-web-projects-in-visual-studio.md). For more information about the authentication options, see [ASP.NET Identity](../identity/overview/index.md).
 
 <a id="scaffold"></a>
 ### ASP.NET Scaffolding
 
 ASP.NET Scaffolding is a code generation framework for ASP.NET Web applications. It makes it easy to add boilerplate code to your project that interacts with a data model.
 
-In previous versions of Visual Studio, scaffolding was limited to ASP.NET MVC projects. With Visual Studio 2013, you can now use scaffolding for any ASP.NET project, including Web Forms. Visual Studio 2013 does not currently support generating pages for a Web Forms project, but you can still use scaffolding with Web Forms by adding MVC dependencies to the project. Support for generating pages for Web Forms will be added in a future update.
+In versions of Visual Studio prior to 2013, scaffolding was limited to ASP.NET MVC projects. Starting in Visual Studio 2013, you can use scaffolding for any ASP.NET project, including Web Forms. Visual Studio does not currently support generating pages for a Web Forms project, but you can still use scaffolding with Web Forms by adding MVC dependencies to the project. Support for generating pages for Web Forms will be added in a future version.
```

Comment:
Is it still accurate that VS doesn't support generating pages for a Web Forms project? @RickAndMSFT